Documentation/installation/k8s-install-helm.rst

```diff
@@ -192,11 +192,17 @@ Install Cilium
 
           Cilium can alternatively run in EKS using an overlay mode that gives
           pods non-VPC-routable IPs.  This allows running more pods per
-          Kubernetes worker node than the ENI limit, but means that pod
-          connectivity to resources outside the cluster (e.g., VMs in the VPC
-          or AWS managed services) is masqueraded (i.e., SNAT) by Cilium to use
-          the VPC IP address of the Kubernetes worker node. To set up Cilium 
-          overlay mode, follow the steps below:
+          Kubernetes worker node than the ENI limit but includes the following caveats:
+
+            1. Pod connectivity to resources outside the cluster (e.g., VMs in the VPC
+               or AWS managed services) is masqueraded (i.e., SNAT) by Cilium to use the
+               VPC IP address of the Kubernetes worker node.
+            2. `Admission webhooks <https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks>`_
```

Comment:
I'd reword this to be more like: "The EKS API Server is unable to route packets to the overlay network. This implies that any webhook which needs to be accessed must be host networked or ...".

The end message I think we should convey is the following:
- there are multiple types of webhooks (e.g., validation, auditing) which are impacted by this
- the EKS API Server is not part of the overlay network -> is unable to communicate with that network